app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -384,12 +295,7 @@ public void onResume() {
         firstLocationUpdate = true;
         locationManager.addLocationListener(this);
 
-        boolean isSettingsChanged = store.getBoolean(Prefs.IS_CONTRIBUTION_COUNT_CHANGED, false);
         store.putBoolean(Prefs.IS_CONTRIBUTION_COUNT_CHANGED, false);
```

Comment:
`Prefs.IS_CONTRIBUTION_COUNT_CHANGED` can be removed entirely, we now only ever set the value, never retrieve

Reply:
Yeah, I think the whole of upload limit setting can be removed now.